dataplane/vppagent/pkg/converter/utils.go

```diff
@@ -26,3 +27,21 @@ func TempIfName() string {
 	logrus.Infof("Generated unique TempIfName: %s len(TempIfName) %d", rv, len(rv))
 	return rv
 }
+
+func TempRouteName() string {
```

Comment:
Question... how do we recover these names to delete the route/arp info on Close() ?

Reply:
On close, interface will be deleted, so I hope it will happen to routes, arp entries, but it will be good idea to have a integration test for this.

Reply:
@haiodo A NSC may have multiple connections to multiple NSEs... so ConnectionContext related to one may need to be deleted, but not all.  Certainly everything goes away when the netns does... but ConnectionContext stuff provisioned needs to be deleted when the Connection is torn down.

Reply:
Have a interface name prefixed routes/arps.